tucson-teststand/support/docker-compose.yaml

```diff
@@ -0,0 +1,96 @@
+version: "3.6"
+
+# The different daemons are meant for different Tucson machines:
+# ospl-daemon1 - ? (master)
+# ospl-daemon2 - ?
+# ospl-daemon3 - ?
+# ospl-daemon4 - ?
+
+services:
+
+  ospl-daemon1:
```

Comment:
Do you want to call it `ospl-master`? or maybe `ospl-main`? since we are ditching `master`.

Reply:
It will be name for whatever machine houses the main daemon like we do for the other sites.